internal/event/event.go

```diff
@@ -14,12 +14,13 @@ type (
 
 // RequestInfo defines the fields captured about a Boundary request.
 type RequestInfo struct {
-	EventId  string `json:"-"`
-	Id       string `json:"id,omitempty" class:"public"`
-	Method   string `json:"method,omitempty" class:"public"`
-	Path     string `json:"path,omitempty" class:"public"`
-	PublicId string `json:"public_id,omitempty" class:"public"`
-	ClientIp string `json:"client_ip,omitempty" class:"public"`
+	EventId  string   `json:"-"`
+	Id       string   `json:"id,omitempty" class:"public"`
+	Method   string   `json:"method,omitempty" class:"public"`
+	Path     string   `json:"path,omitempty" class:"public"`
+	PublicId string   `json:"public_id,omitempty" class:"public"`
+	ClientIp string   `json:"client_ip,omitempty" class:"public"`
+	Actions  []string `json:"actions,omitempty" class:"public"`
```

Comment:
Do we need to add this? This will add this line to every event we send, right? Perhaps we could parse the actions out of the path in the handler instead?

Reply:
Yeah we actually do not need this added to eventing anymore. That was part of another implementation where I wanted to use the RequestInfo event to set the actions